client-app/core/api/graphql/client.ts

```diff
@@ -1,41 +1,13 @@
-import { ApolloClient, InMemoryCache } from "@apollo/client/core";
-import { cache, deprecatedLink, link, options } from "@/core/api/graphql/config";
-
-const fetchPolicy = "no-cache";
-
-/**
- * Non-cached version of Apollo Client
- *
- * @deprecated
- *
- * Use {@link apolloClient} instead.
- *
- * **IMPORTANT!** Because of cache usage, query deduplication and `@vue/apollo-composable` usage,
- * you can't just replace {@link graphqlClient} with {@link apolloClient} in your code.
- * Look at https://github.com/VirtoCommerce/vc-theme-b2b-vue/pull/868 for more information.
- */
-export const graphqlClient = new ApolloClient({
-  ...options,
-  link: deprecatedLink,
-
-  cache: new InMemoryCache({
-    addTypename: false,
```

Comment:
`addTypename`  will slightly influence the volume of network traffic, because requests and responses include __typename field. But this increasing can be considered negligible.